doc/UsersGuide/CustomizingTheWorkflow/InputOutputFiles.rst

```diff
@@ -20,8 +20,9 @@ The external model files needed for initializing an experiment can be obtained i
 ways, including: 
 
    * Pulled from the `SRW App Data Bucket <https://registry.opendata.aws/noaa-ufs-shortrangeweather/>`__,
-   * Pulled from the NOAA High Performance Storage System (:term:`HPSS`) during the workflow execution (requires user access), or 
-   * Obtained and staged by the user from a different source. 
+   * Pulled from the NOAA High Performance Storage System (:term:`HPSS`) during the workflow execution (requires user access), 
+   * Obtained and staged by the user from a different source, or 
+   * Pulled from the `RRFS data bucket (rrfs_a data) <https://registry.opendata.aws/noaa-rrfs/>`_.
```

Comment:
I think this section is more general and the mention of RRFS specifically should not go here.

Reply:
This is included specifically because the new test case for the RRFS is not staged in standard locations, as many other WE2E experiments (data pulled from the SRW App Data Bucket). It is also a ready reference for obtaining readily available initialization data besides the HPSS or other sources ("Obtained and staged by the user from a different source"), for users external to NOAA who may not be aware of actual data sources.
The question of where to get the data to initialize the UFS/SRW model is one of the most frequently asked questions (sometimes directly sent to Maoyi), so it looks that including this info in the section on "Input and Output" is likely very beneficial to users.